code/drasil-docLang/lib/Drasil/DocumentLanguage/TraceabilityGraph.hs

```diff
@@ -162,10 +162,13 @@ traceGLst = UlC $ ulcc $ Enumeration $ Bullet $ map (, Nothing) folderList'
 
 -- | The Traceability Graph contents.
 traceGCon :: String -> [Contents] -- FIXME: HACK: We're generating "LlC"s of the traceability graphs multiple times... See DocumentLanguage.hs' mkTraceabilitySec for the other spot.
-traceGCon ex = map LlC (zipWith (traceGraphLC ex) traceGFiles traceGUIDs)
+traceGCon ex = map LlC (traceGCon' ex)
             ++ [mkParagraph $ S "For convenience, the following graphs can be\
                \ found at the links below:", traceGLst]
 
+traceGCon' :: String -> [LabelledContent]
+traceGCon' ex = zipWith (traceGraphLC ex) traceGFiles traceGUIDs
```

Comment:
The 'real fix' should be to have all internally-required `LabelledContent` chunks handled as any other 'required chunks'. i.e., to include them in the basic `ChunkDB`. Note recent discussion on #4384 .

Reply:
I added a little bit more discussion in abebc93467840848c50bbdc4aeb67c00a33b4b9a . This common function is what generates the example-specialized copy of the traceability graphs. See the Template's `.drasil-requirements.csv` diff, for example, which contains the `progname`. For the other examples, it's their actual codenames.